JSDemos/Demos/TreeView/LoadDataOnDemand/description.md

```diff
@@ -1 +1,9 @@
-In&nbsp;those special cases when the built-in Virtual Mode fails, you can gain full control over data loading in&nbsp;the **createChildren** function.This function is&nbsp;called each time a&nbsp;user expands the TreeView node that has not been expanded before. In&nbsp;the **createChildren** function, you get data from a&nbsp;server or&nbsp;other sources and then form the collection of&nbsp;child nodes from this data.
\ No newline at end of file
+Use the [createChildren](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#createChildren) function to load data on demand and enhance performance on large datasets. Unlike virtual mode, this function allows you to use your own custom logic. 
+
+To specify the function, pass the expanded node's identifier as an argument and process the returned value inside the function. This value can be an array of objects or a promise as in this demo. The following prerequisites should be met:
```

Comment:
>To specify the function,

You do not want to specify the function but enable some behavior, right?

>pass the expanded node's identifier as an argument and process the returned value inside the function.

When a function returns a value, the function's execution is over, isn't it (I really just wondering, I might be wrong)? How about the following:
To enable this behavior, pass the the expanded node's data to the **createChildren** function and process this data in it.

>The following prerequisites should be met:

I'm not sure that the word 'prerequisites' is the best choice. I would suggest keeping the sentence from the createChildren description: 'This function has the following restrictions:'